.github/workflows/ci.yml

```diff
@@ -24,6 +24,10 @@ jobs:
         run: python -m pip install --upgrade nox pip
       - name: Run linters
         run: nox -vs lint
+      - name: Validate changelog
+        uses: zattoo/changelog@v1
```

Comment:
v1.1.0 is out since June 22. Other than that LGTM, retarget

Reply:
It's not officially released, unavailable in the marketplace, and has only one additional feature that we don't need:
> Support glob in sources